cmd/release/README.md

```diff
@@ -0,0 +1,47 @@
+# Release
+
+**Release** is a command line tool that automates the versioning and releasing of multi-module Go projects.
+
+## Limitations
+
+- All modules have the same version.
```

Comment:
I think we should solve this limitation for sure, I have observed this to be a problem often. Specially with auto-dependency bump tools like renovate.

Reply:
Ideally yes. I’m still unsure about how to go about solving this